Makefile

```diff
@@ -152,3 +152,27 @@ runserver: ## Run the Django application locally
 
 .PHONY: dev-init
 dev-init: load-design-system-templates collectstatic makemigrations migrate createsuperuser ## Run the pre-run setup scripts
+
+.PHONY: functional-tests-up
+functional-tests-up:  ## Start the functional tests docker compose dependencies
+	docker compose -f functional_tests/docker-compose.yml up -d
+
+.PHONY: functional-tests-up-dev
+functional-tests-up-dev:  ## Start the functional tests docker compose dependencies and dev app
```

Comment:
I think these are the wrong way around. `up` should be all, and `dev` should be backing services.

Reply:
In this case I thought this way made more sense, since the app is only needed if the developer wants it, for manually interacting with in development, but I can change it for consistency.